lib/rubocop/cop/lint/duplicate_methods.rb

```diff
@@ -132,8 +132,18 @@ def message_for_dup(node, method_name)
                       current: source_location(node))
         end
 
+        def found_sclass_method(node, name)
```

Comment:
That's just my two cents. Can you move this `found_sclass_method` method definition between `found_instance_method` and `found_method`  method definitions?

Reply:
Makes sense indeed, just did